src/stores/__tests__/torrent-store.spec.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import lodash from 'lodash';
+import TorrentStore from '../torrent-store'
+import Torrent from '../torrent'
+
+const {
+  STATUS_STOPPED,
+  STATUS_CHECK_WAIT,
+  STATUS_CHECK,
+  STATUS_DOWNLOAD_WAIT,
+  STATUS_DOWNLOAD,
+  STATUS_SEED_WAIT,
+  STATUS_SEED
+} = Torrent;
+
+const mapStatus = arr => lodash.map(arr, 'status')
```

Comment:
I thought lodash was used throughout the project (didn't realize it only had one reference!) I defintely agree that we should keep it out :)